### ISSUE TITLE
Detailed descriptions in alt text on main README.md

### ISSUE THREAD

**Issue by brooke-hamilton** [Azure/missionlz#601]
## Benefit/Result/Outcome

So that a person using a screen reader will get useful information from the charts on the root README.md file.

## Description

The charts shown as images on the main page README.md file have very short alt text descriptions that would not be useful to a person using a screen reader. Update the descriptions to be more useful.

## Acceptance Criteria

-  All png and svg files have alt text that fully describes the content of the images.
-  Each alt text is 250 characters or less.

**Comment by glennmusa** [comment]
On the topic of accessibility, could we make the background for graphics a solid color instead of transparent? Today, some of the diagrams lack contrast with dark elements on transparent backgrounds in GitHub dark themes.

**Comment by brooke-hamilton** [comment]
Which solid color do you think would work with both themes?

**Comment by glennmusa** [comment]
Black body on white backgrounds is probably the most universal and is what have been publishing to docs.azure.com, but if we wanted to get fancy: https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to
